backend/tests/clubs/test_commands.py

```diff
@@ -651,3 +651,46 @@ def test_wrong_arguments(self):
 
         with self.assertRaises(CommandError):
             call_command("merge_duplicates", "--tag")
+
+
+class ExpireMembershipInvitesTest(TestCase):
+    def setUp(self):
+        # Users
+        self.user1 = get_user_model().objects.create_user(
+            "bfranklin", "bfranklin@seas.upenn.edu", "test"
+        )
+        self.user2 = get_user_model().objects.create_user(
+            "tjefferson", "test-application-notif@example.com", "test"
+        )
+
+        # Club
+        self.club1 = Club.objects.create(
+            code="one", name="Club One", active=True, email="test@example.com"
+        )
+
+        # Expired MembershipInvite
```

Comment:
nit: comments could be more descriptive. Something like "Create expired membership invite" is generally better practice.